dask_jobqueue/sge.py

```diff
@@ -82,3 +84,42 @@ def __init__(self, queue=None, project=None, resource_spec=None, walltime=None,
         self.job_header = header_template % config
 
         logger.debug("Job script: \n %s" % self.job_script())
+
+    def start_workers(self, n=1, ta=True):
```

Comment:
I know this is only a poc on SGE, but I would propose a `use_job_arrays`, or `use_task_arrays` at class level. So directly in JobQueueCluster `__init__()` method in the end.

Reply:
Hi @guillaumeeb ,
this is a nice suggestion. I can implement the change.
I am working on this in my free time, so I am quite slow to follow up...

Reply:
Sure, take your time, I know this situation :)